TeachEasy/src/teacheasy/data/AudioData.java

```diff
@@ -0,0 +1,44 @@
+package teacheasy.data;
+
+/**
+ * A class to hold audio data
+ * @author sr896 sh1157
+ *
+ */
+public class AudioData extends PageObject {
+
+	boolean viewProgress;
+	String uri;
+	String url;
+	
+	/**Constructor Method*/
+	public AudioData (PageObjectType nType, float nXStart, float nYStart) {

```

Comment:
All class variables should be instantiated by the constructor to avoid null pointer exceptions. viewProgress, uri and url should have corresponding parameters in the constructor to set their initial values.